### PR TITLE
[IMP] fieldservice_mobile: Record rule Portal Partner multi-company

### PR DESCRIPTION
**Pull request by murtuzasaleh**
1) Added Record Rule because of the portal user does not have res.partner access and we are not getting Location and Partner fields in Mobile.

2) https://wiki.opensourceintegrators.com/art/index.php?title=FSM_Mobile_for_contractors

### CODE REVIEW

File: fieldservice_mobile/__manifest__.py [hunk 1]
```diff
@@ -7,14 +7,15 @@
...
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.1.0',
     'category': 'Field Service',
     'author': 'Open Source Integrators',
     'website': 'https://github.com/ursais/osi-addons',
     'depends': [
         'fieldservice_stage_server_action',
     ],
     'data': [
+        'security/partner_security.xml',
```
Comment (max3903) [inline fieldservice_mobile/__manifest__.py]:
```suggestion
        'security/ir_rule.xml',
```

**Comment by max3903** [comment]
@murtuzasaleh This will allow a portal user to see all the contacts of the company. I am not sure we want to do this...

**Comment by murtuzasaleh** [comment]
> contacts

@max3903 But when we logged in with a portal user in mobile we can not get partner_latitude, partner_longitude, and other (Location and Partner fields) Because of this record rule [res_partner: portal/public: read access on my commercial partner](https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/security/base_security.xml#L27)

Any solution to this issue?
unnamed

File: fieldservice_mobile/security/partner_security.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<odoo noupdate="1">
+
+    <record id="portal_partner_comp_rule" model="ir.rule">
+        <field name="name">Portal Partner multi-company</field>
+        <field name="model_id" ref="base.model_res_partner"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
```
Comment (max3903) [inline fieldservice_mobile/security/partner_security.xml]:
```suggestion
        <field name="domain_force">['&', ('is_fsm_location', '=', True), '|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
```

**Review by smangukiya — APPROVED** [review]

**Pull request merged**